.circleci/config.yml

```diff
@@ -53,6 +53,9 @@ executors:
   golang-ci:
     docker:
       - image: okteto/golang-ci:1.18.0
+  golang-ci-new:
+    docker:
+      - image: okteto/golang-ci:2.2.2
```

Comment:
Should We update all the references to `golang-ci 2.2.2`?

Reply:
I'll check, the `2.2.2` could have some breaking changes with `jq` and `yq`

Reply:
@jLopezbarb no problem by updating!